backend/src/schema/resolvers/reports.spec.js

```diff
@@ -66,70 +73,98 @@ describe('report', () => {
+                    type: null,
+                  })
+                })
+        */
         it('returns the submitter', async () => {
           returnedObject = '{ submitter { email } }'
           await expect(action()).resolves.toEqual({
-            report: { submitter: { email: 'test@example.org' } },
+            report: {
+              submitter: {
+                email: 'test@example.org',
+              },
+            },
           })
         })
 
         describe('reported resource is a user', () => {
           it('returns type "User"', async () => {
             returnedObject = '{ type }'
             await expect(action()).resolves.toEqual({
-              report: { type: 'User' },
+              report: {
+                type: 'User',
+              },
             })
           })
 
           it('returns resource in user attribute', async () => {
             returnedObject = '{ user { name } }'
             await expect(action()).resolves.toEqual({
-              report: { user: { name: 'abusive-user' } },
+              report: {
+                user: {
+                  name: 'abusive-user',
+                },
+              },
             })
           })
         })
 
         describe('reported resource is a post', () => {
           beforeEach(async () => {
-            await factory.authenticateAs({ email: 'test@example.org', password: '1234' })
+            await factory.authenticateAs({
+              email: 'test@example.org',
+              password: '1234',
+            })
             await factory.create('Post', {
               id: 'p23',
               title: 'Matt and Robert having a pair-programming',
             })
-            variables = { id: 'p23' }
+            variables = {
+              id: 'p23',
+            }
           })
 
           it('returns type "Post"', async () => {
             returnedObject = '{ type }'
             await expect(action()).resolves.toEqual({
-              report: { type: 'Post' },
+              report: {
+                type: 'Post',
+              },
             })
           })
 
           it('returns resource in post attribute', async () => {
             returnedObject = '{ post { title } }'
             await expect(action()).resolves.toEqual({
-              report: { post: { title: 'Matt and Robert having a pair-programming' } },
+              report: {
+                post: {
+                  title: 'Matt and Robert having a pair-programming',
+                },
+              },
             })
           })
 
           it('returns null in user attribute', async () => {
             returnedObject = '{ user { name } }'
             await expect(action()).resolves.toEqual({
-              report: { user: null },
+              report: {
+                user: null,
+              },
             })
           })
         })
 
+        /* An der Stelle würde ich den p23 noch mal prüfen, diesmal muss aber eine error meldung kommen.
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T12:49:44Z" title="Friday, June 21st 2019, 2:49:44 pm +02:00">Jun 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -66,70 +73,98 @@ describe('report', () => {
 
       describe('valid resource id', () => {
         beforeEach(async () => {
-          variables = { id: 'u2' }
-        })
-
-        it('creates a report', async () => {
-          await expect(action()).resolves.toEqual({
-            report: { description: 'Violates code of conduct' },
-          })
+          variables = {
+            id: 'u2',
+          }
         })
-
+        /*
+                it('creates a report', async () => {
+                  await expect(action()).resolves.toEqual({
+                    type: null,
+                  })
+                })
+        */
         it('returns the submitter', async () => {
           returnedObject = '{ submitter { email } }'
           await expect(action()).resolves.toEqual({
-            report: { submitter: { email: 'test@example.org' } },
+            report: {
+              submitter: {
+                email: 'test@example.org',
+              },
+            },
           })
         })
 
         describe('reported resource is a user', () => {
           it('returns type "User"', async () => {
             returnedObject = '{ type }'
             await expect(action()).resolves.toEqual({
-              report: { type: 'User' },
+              report: {
+                type: 'User',
+              },
             })
           })
 
           it('returns resource in user attribute', async () => {
             returnedObject = '{ user { name } }'
             await expect(action()).resolves.toEqual({
-              report: { user: { name: 'abusive-user' } },
+              report: {
+                user: {
+                  name: 'abusive-user',
+                },
+              },
             })
           })
         })
 
         describe('reported resource is a post', () => {
           beforeEach(async () => {
-            await factory.authenticateAs({ email: 'test@example.org', password: '1234' })
+            await factory.authenticateAs({
+              email: 'test@example.org',
+              password: '1234',
+            })
             await factory.create('Post', {
               id: 'p23',
               title: 'Matt and Robert having a pair-programming',
             })
-            variables = { id: 'p23' }
+            variables = {
+              id: 'p23',
+            }
           })
 
           it('returns type "Post"', async () => {
             returnedObject = '{ type }'
             await expect(action()).resolves.toEqual({
-              report: { type: 'Post' },
+              report: {
+                type: 'Post',
+              },
             })
           })
 
           it('returns resource in post attribute', async () => {
             returnedObject = '{ post { title } }'
             await expect(action()).resolves.toEqual({
-              report: { post: { title: 'Matt and Robert having a pair-programming' } },
+              report: {
+                post: {
+                  title: 'Matt and Robert having a pair-programming',
+                },
+              },
             })
           })
 
           it('returns null in user attribute', async () => {
             returnedObject = '{ user { name } }'
             await expect(action()).resolves.toEqual({
-              report: { user: null },
+              report: {
+                user: null,
+              },
             })
           })
         })
 
+        /* An der Stelle würde ich den p23 noch mal prüfen, diesmal muss aber eine error meldung kommen.
```


```suggestion
        /*
```

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T12:51:35Z" title="Friday, June 21st 2019, 2:51:35 pm +02:00">Jun 21, 2019</time>_

---


You can expect errors in a test like this:
```
expect(action()).rejects.toThrow("Some error message')
```

See jest's excellent documentation: https://jestjs.io/docs/en/expect.html#rejects